<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contourFilterPropForm</class>
 <widget class="QWidget" name="contourFilterPropForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contour Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>contour by</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="val_spinbox">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="reflection_checkBox">
     <property name="text">
      <string>Reflection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="reflection_groupBox">
     <property name="title">
      <string>reflection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>reflection:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="reflection_comboBox">
        <item>
         <property name="text">
          <string>X_MIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y_MIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z_MIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X_MAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y_MAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z_MAX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>117</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
